mh/state.py

```diff
@@ -457,11 +641,30 @@ def get_game_time(self):
         pass
 
     def get_servers(self):
-            return self.servers
+        if not self.cache:
+            return []
+        server_list = self.cache.get_server_list()
+        if self.server_id == 0:
+            return server_list
+        below, above = [], []
+        i = 0
+        for server in server_list:
+            i += 1
+            if i < self.server_id:
+                below.append(server)
+            elif i > self.server_id:
+                above.append(server)
```

Comment:
Can't the server list be returned sorted?

Reply:
This function was, at the time of this review, incorrectly implemented due to a conflation between "index" and "server_id." It has since been corrected. It now sorts by `server_id` according to the Cache's records of what each server's ID is.